src/plasma/plasma_manager.cc

```diff
@@ -464,7 +458,7 @@ PlasmaManagerState *PlasmaManagerState_init(const char *store_socket_name,
     db_connect_args[1] = store_socket_name;
     db_connect_args[2] = "manager_socket_name";
     db_connect_args[3] = manager_socket_name;
-    db_connect_args[4] = "address";
+    db_connect_args[4] = "aux_address";
```

Comment:
Mostly so that we can have just one field for both the local scheduler and the plasma manager, instead of two. I guess it's not totally necessary, but it makes the parsing on the Redis module a little easier if we only have to look for one string. I think calling it `manager_address` instead probably makes sense though.